frontend/src/types.ts

```diff
@@ -605,38 +605,68 @@ export enum ProgressionStep {
   SERVER_REQUESTED = 'SERVER_REQUESTED',
   POD_CREATED = 'POD_CREATED',
   POD_ASSIGNED = 'POD_ASSIGNED',
+  POD_PROBLEM = 'POD_PROBLEM',
   PVC_ATTACHED = 'PVC_ATTACHED',
   INTERFACE_ADDED = 'INTERFACE_ADDED',
   PULLING_NOTEBOOK_IMAGE = 'PULLING_NOTEBOOK_IMAGE',
   NOTEBOOK_IMAGE_PULLED = 'NOTEBOOK_IMAGE_PULLED',
   NOTEBOOK_CONTAINER_CREATED = 'NOTEBOOK_CONTAINER_CREATED',
+  NOTEBOOK_PROBLEM = 'NOTEBOOK_PROBLEM',
```

Comment:
```suggestion
  NOTEBOOK_CONTAINER_PROBLEM = 'NOTEBOOK_CONTAINER_PROBLEM',
```
Sorry for misleading you -- looks better if you just mirror the other naming conventions


(do the same for OAuth Problem)